RELEASENOTES/v2.0.0.alpha-4.MD

```diff
@@ -0,0 +1,56 @@
+    - Signature format (JWS) for signing and verification in RC-1.
+    - Updates on use of plugins for signing and verification. Plugins let vendors and users integrate base Notation client with their choice of Key Vaults and PKI.
+ - Update "Notation" (CLI client) implementation to match the latest specifications. With the new client, users can
+  - Besides the features from Alpha-3, this releases adds support for "self signed certificates" for use in non production use cases
+
+*We do not recommend this release for use in production environment. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). We suggest using it for testing and providing feedback to the Notary v2 working group only*
+
+## List of feature/capabilities added with this release
+For detailed notes on each of the individual repositories (NotaryProject, Notation (for reference CLI implementation) and Notation libraries) users can refer to the individual release notes. A summary view is provided in this document.
+
+### Notary v2 specifications
+Updated Trust Policy, Trust Store,  Signature Verification and Signing workflows along with extensibility mechanisms via Notation plugins.
+- A final JWS based [signature envelope format.](https://github.com/notaryproject/notaryproject/blob/main/signature-envelope-jws.md)
+- Updated [Signature specfication](https://github.com/notaryproject/notaryproject/blob/main/signature-envelope-jws.md) with extensibility provisions for additional signature envelope format.
+- Updated [signing and verification workflows](https://github.com/notaryproject/notaryproject/blob/main/signing-and-verification-workflow.md)
+- Introducing a new [Signing scheme](https://github.com/notaryproject/notaryproject/blob/main/signing-scheme.md) to integrate Notation Client with Signing Services
+- Updated  [trust store and trust policy](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md) specification
+
+### Notation CLI Client
+Notation is a standalone CLI client based on Notary v2 specs. It supports signing, verifying, and storing signatures in [oras-artifacts-spec](https://github.com/oras-project/artifacts-spec/) compliant registries. Starting with this release of Notation, the trust store will be based on a directory structure and trust policy as a JSON document. Users can simply insert  the public certificates inside pre-configured directories for configuring their trust store, and modify the trust policy document to granularly control the scope and behavior of the signature verification policies. The use of the existing CLI commands (introduced in the alpha-1 version of Notation client when the keys/certificates for signing and verifying were specified in a file) has been subsituted by use of a directory based trust store.  Refer https://github.com/notaryproject/notation/blob/main/specs/notation-config.md.
```

Comment:
As per conversation with Vani and Yi, trust store and trust policy is moved out of alph4 to rc1. So we shouldn't add this.

Reply:
@priteshbandi @iamsamirzon  I think the spec and Go SDK implementation of the trust store & trust policy is ready now, but the capability of managing the trust store and trust policy with CLI is not available in alpha.4. 

This part was still planned in RC.1 and there is a [PR](https://github.com/notaryproject/notation/pull/361/files#diff-a90f3a9bb851323205a06305621b96bde0e0603ea366e389c363ecefc9858d82R37) to illustrate the CLI behavior. So I would suggest removing the trust store & trust policy from this paragraph, or we can mention it in the Notation library below.

Reply:
Thanks @priteshbandi for clarifying this. The TS/TP managementation is completed in notation-go library. Although users can manually configure TS/TP, but the user experience will be cumbersome.  I still suggest we don't explicitly tell users about the TS/TP in the Notation CLI Client section and move this to the Notation Library below.

Reply:
@priteshbandi @vaninrao10 @vaninrao @dtzar  Please check this issue https://github.com/notaryproject/roadmap/issues/38, the verify command using trust store and trust policy is part of this user story, and it is planned in rc.1, not alpha.4. If we want to include verify command using TS/TP, we also need to provide a guide to show user how to create trust store and trust policy, we also need to update hello-sign.md and other existing E2E document for it. These issues are in rc-1 scope. This is the PR for Trust store CLI spec https://github.com/notaryproject/notation/pull/361, and I am also working on verify CLI spec for using TS/TP.

Here is my proposal: 
- Continue to release alpha.4 for this issue "relax certificate chain requirement for self-signed certificate."
- Update [roadmap 45 ](https://github.com/notaryproject/roadmap/issues/45) and release notes to reflect the TS/TP supported in library only.

Reply:
Agree we need to provide 1. Guide to the users to create TS / TP. 2. Update hello-sign.md 3. Other existing E2E document,
this will give a good feature push to Alpha-4 and we will be able to collect the feedback from DA. So, prior to the launch / RC-1 we will be able to act on the feedback. I think its a win - win scenario.

Reply:
Updated the Alpha-4 system release notes to reflect that Notation does not directly support CLI commands for managing trust store and trust policy